lib/global_uid/base.rb

```diff
@@ -29,169 +20,104 @@ def self.create_uid_tables(id_table_name, options={})
       type     = options[:uid_type] || "bigint(21) UNSIGNED"
```

Comment:
IMO, `create_uid_tables` and  `drop_uid_tables ` should be moved into another class, the two table operations have nothing to do with the ID allocation code below, they're here for the convenience of accessing `with_servers`